packages/lerna-changelog-generator/src/index.ts

```diff
@@ -106,8 +112,14 @@ const generateChangelog = async () => {
     console.warn('No new versions have been released');
     return 'false';
   }
+
+  const argv = await yargs(hideBin(process.argv)).argv;
   const packagesWithChangedDeps = findChangedDependencies(allFiles);
-  writeChangedDependencies(packagesWithChangedDeps);
+  writeChangedDependencies({
+    packages: packagesWithChangedDeps,
+    currentProjectUrl: (argv.projectUrl as string) || process.env['CI_PROJECT_URL'] || '',
+    defaultBranch: (argv.defaultBranch as string) || process.env['CI_DEFAULT_BRANCH'] || '',
```

Comment:
nit: что если `argv` === `undefined`?
```suggestion
    currentProjectUrl: (argv?.projectUrl as string) || process.env['CI_PROJECT_URL'] || '',
    defaultBranch: (argv?.defaultBranch as string) || process.env['CI_DEFAULT_BRANCH'] || '',
```

Reply:
Тип у него описан как обязательный объект с полями never, так что вероятно это не нужно.